README.md

```diff
@@ -1,199 +0,0 @@
-## charts
```

Comment:
Can you add a very basic README that says go to `dev-v2.5-source`, we don't merge PRs here.